backend/app/routes/admin/getAdmins.js

```diff
@@ -0,0 +1,53 @@
+const to = require('await-to-js').default;
+const Admin = require('../../models/Admin');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+const createPipeline = (adminType, page) => {
+  const pipeline = [
+    {
+      $project: {
+        _id: 0,
+        firstName: '$firstName',
+        lastName: '$lastName',
+        username: '$username',
+        email: '$email',
+        contact: '$contact',

```

Comment:
Add isSuperAdmin also since if the query doesnt contain any type then we will display all results and one should be able to identify the type looking at the response